sources/packages/backend/apps/api/test/jest-e2e.json

```diff
@@ -25,5 +25,21 @@
     "@sims/test-utils": "<rootDir>/../../libs/test-utils/src",
     "@sims/integrations/(.*)": "<rootDir>/../../libs/integrations/src/$1",
     "@sims/integrations": "<rootDir>/../../libs/integrations/src"
-  }
+  },
+  "coveragePathIgnorePatterns": [
+    "node_modules",
```

Comment:
mm. I have this question. how does jest identify if it's a file or a folder, for example in I have a folder `x/` and a file in another folder called `x.ts`.  and I need only to exclude `x/` folder not the `x.ts` file. how will we configure that?

Reply:
> For example adding coveragePathIgnorePatterns:["test"] ignored all the files in the test because the file path names are
> 
> ```
> test/file1
> test/folder2/file1
> test/file2
> ```
> 

mm.. so if there is a file called `test.ts`, it will also be ignored, in this example right?

Reply:
Yes, test.ts will be ignored.